sparse_strips/vello_hybrid/shaders/sparse_strip_renderer.wgsl

```diff
@@ -0,0 +1,81 @@
+// Copyright 2024 the Vello Authors
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+
+// A WGSL shader for rendering sparse strips with alpha blending.
+//
+// Each strip instance represents a horizontal slice of the rendered output and consists of:
+// 1. A variable-width region of alpha values for semi-transparent rendering
+// 2. A solid color region for fully opaque areas
+//
+// The alpha values are stored in a texture and sampled during fragment shading.
+// This approach optimizes memory usage by only storing alpha data where needed.
+
+struct Config {
+    width: u32,
+    height: u32,
+    strip_height: u32,
+    alpha_texture_width: u32,
```

Comment:
I think you might be able to use [`textureDimensions`](https://www.w3.org/TR/WGSL/#texturedimensions) to get the size of this texture.